### ISSUE TITLE
scheduler doesn't provide right route outside of app.py

### ISSUE THREAD

**Issue by anovis** [aws/chalice#1016]
I created a scheduler route in `chalicelib` along my other routes and imported Chalice from app.py. The regular routes work, however the lambda that was created for the scheduler had the handler as `app.route` where it should have been `app.chalicelib.route`.

**Comment by stealthycoin** [comment]
Correct. Routes only go in `app.py` which is, at least for now, by design and on purpose. Looking through the docs its not super clear though so I can understand the confusion. Ill mark this as the docs needing to be clearer.

**Comment by stealthycoin** [comment]
It looks like once blueprints (#1023) are merged into an experimental "thing" (not clear, waiting on the outcome of #1019) this will no longer be the case. So I think i'll change this to feature request for now.
